node_modules/oae-core/about/about.html

```diff
@@ -0,0 +1,48 @@
+<!-- CSS -->
```

Comment:
There's already an `about<resourceType>` widget for every resource type. There is nothing that strictly makes this for groups, except for some message labels that are in there for simplicity while group is the only consumer of the widget.